readme.md

```diff
@@ -25,8 +25,8 @@ ___
 
 ❗️ Replace `<your_account>` with your Github username and copy the links to `Pull Request` description:
 
-- [DEMO LINK](https://<your_account>.github.io/layout_search-bar-airbnb/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_search-bar-airbnb/report/html_report/)
+- [DEMO LINK](https://IrenaPopovych7.github.io/layout_search-bar-airbnb/)
+- [TEST REPORT LINK](https://IrenaPopovych7.github.io/layout_search-bar-airbnb/report/html_report/)
```

Comment:
Add these links to the PR description
<img width="799" alt="image" src="https://github.com/mate-academy/layout_search-bar-airbnb/assets/64750958/0a51982e-ff33-4c60-acf8-9fb6125e9a27">

Reply:
Still not added
<img width="985" alt="image" src="https://github.com/mate-academy/layout_search-bar-airbnb/assets/64750958/9272bd34-6396-4640-a80e-8d18ea757711">